### PR TITLE
Made tutorial creation form

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello link : https://trello.com/c/r3R3GbQC/43-make-tutorial-creation-form

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -48,6 +50,7 @@ class Routes extends Component {
           <ProtectedRoute path={JAVASCRIPT} component={Javascript} />
           <ProtectedRoute path={CODE} component={Code} />
           <ProtectedRoute path={PROFILE} component={Profile} />
+          <GuestRoute path={TUTORIAL_FORM} component={TutorialForm} />
```
Comment (ayousaf11) [inline src/routes/index.js]: @HarisShahidHameed 
Changes this `GuestRoute` into a `ProtectedRoute`

File: src/containers/Dashboard/Forms/TutorialForm.js [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+                     "required"
+                   )}
+                 </div>
+                </div>
+                <div>
+                  <label className="labels">
+                    Tags
+                  </label>
+                </div>
+                {this.state.tags.map((tag, idx) => (
```
Comment (ayousaf11) [inline src/containers/Dashboard/Forms/TutorialForm.js]: @HarisShahidHameed 
use meaningful `variables` names. Replace `idx` here

File: src/containers/Dashboard/Forms/TutorialForm.js [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    e.preventDefault();
+    if (this.validator.allValid()) {
+      var { name, content, courseId, tags } = this.state;
+      const data = {
+        name,
+        content,
+        courseId,
+        tags
+      };
+      debugger;
```
Comment (ayousaf11) [inline src/containers/Dashboard/Forms/TutorialForm.js]: @HarisShahidHameed 
you should not commit any `debugger`

File: src/routes/index.js (hunk 1 above)
Comment (HarisShahidHameed) [reply src/routes/index.js]: Done

File: src/containers/Dashboard/Forms/TutorialForm.js (hunk 2 above)
Comment (HarisShahidHameed) [reply src/containers/Dashboard/Forms/TutorialForm.js]: Done. Changed it to index

File: src/containers/Dashboard/Forms/TutorialForm.js (hunk 3 above)
Comment (HarisShahidHameed) [reply src/containers/Dashboard/Forms/TutorialForm.js]: Removed.

**Pull request merged**